src/cli/cmds/synth.ts

```diff
@@ -18,7 +26,16 @@ class Command implements yargs.CommandModule {
     .option('stdout', { type: 'boolean', required: false, desc: 'Write synthesized manifests to STDOUT instead of the output directory', alias: 'p' })
     .option('plugins-dir', { default: config.pluginsDirectory, required: false, desc: 'Directory to store cdk8s plugins.' })
     .option('validate', { type: 'boolean', default: true, required: false, desc: 'Apply validation plugins on the resulting manifests (use --no-validate to disable)' })
-    .option('validation-reports-output-file', { required: false, desc: 'File to write a JSON representation of the validation reports to' });
+    .option('validation-reports-output-file', { required: false, desc: 'File to write a JSON representation of the validation reports to' })
+    .option('format', {
+      default: config.synth!.format,
```

Comment:
Avoid using `!` unless absolutely necessary. Here - I believe it isn't? (make sure to address other places that use this as well)